### ISSUE TITLE
let ptline-output write a valid routeFile

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#9726]
write flows with routes and a single vehicle per line, all attribute that do not fit a flow are written as `<param>`

advantages:
- output can be loaded directly to investigate routes without running ptlines2flows
- ptlines can be edited with netedit
- ptlines2flows can be applied to regular route files which makes it easiert to generate synthetic schedules for non-osm scenarios

**Comment by behrisch** [comment]
I still remember that we discussed this internally whether we need a new file format when we invented the ptline output. Is the change of mind mainly for netedit support?

**Comment by namdre** [comment]
Yes. I remember the discussion all to well. My renewed thinking on the issue was prompted by the first of the three points listed above but I think all three points weigh equally strong.
Also, back then it was still unclear how much extra features we would need to support for ptLines (which I didn't want to hoist onto the already-complicated route file format. By now I'm more confident that I understand that range of features.
